docusaurus/docs/dev-docs/migration/v4-to-v5/additional-resources/plugins-migration.md

```diff
@@ -1,35 +1,50 @@
 ---
-title: Plugins upgrade summary
+title: Plugins upgrade
 displayed_sidebar: devDocsMigrationV5Sidebar
 tags:
   - plugins
   - plugins development
   - upgrade to Strapi 5
 ---
 
-# Plugins upgrade summary
+# Plugins upgrade
 
 The present page is intended to be used as a short summary of everything to consider if you are a plugin developer intending to upgrade your plugin from Strapi v4 to Strapi 5. The page quickly describes the changes affecting plugins and links to additional resources where necessary.
 
+:::caution
+Starting with Strapi 5, the platform includes a [Plugin SDK](https://github.com/strapi/sdk-plugin) to help develop and build Strapi plugins. While you are free to use other methods for plugin development, we strongly recommend using the Plugin SDK. As part of the migration process from v4 to v5, we encourage you to update your plugin to take advantage of the Plugin SDK.
+:::
+
+## Recommended steps to migrate a plugin
+:::prerequisites
+Your Strapi application is already running on the latest v5 minor and patch version. If it's not, follow the [step-by-step guide](/dev-docs/step-by-step) to upgrade to Strapi 5.
+:::
+
 Upgrading a Strapi v4 plugin to Strapi 5 consists in:
 
 1. Creating a new empty plugin using the [Plugin SDK](/dev-docs/plugins/development/create-a-plugin).
 2. Move your Strapi v4 code to the newly created files in the Strapi 5 [plugin structure](/dev-docs/plugins/development/plugin-structure), also considering the changes summarized in this page.
 
-## Back-end changes
+Alternatively, you can manually update your Strapi v4 plugin to use the Plugin SDK.
+The manual steps include:
+1. If your code uses a format other than CommonJS, update the `package.json` file and specify the appropriate exports property.
```

Comment:
I'm not sure it's enough to say this way
@alexandrebodin wdyt?

Reply:
You can go ahead with this I'll update it in my PR 👍